stm32cube/stm32f3xx/README

```diff
@@ -35,15 +35,4 @@ License Link:
    https://opensource.org/licenses/BSD-3-Clause
 
 Patch List:
-
-   *Changes from official delivery:
```

Comment:
This should be kept

Reply:
yes, patch list remains and includes a warning 

> _please check that the following list is still valid_

plus a link to the new _release_note.html from STM32Cube_